src/snapred/backend/data/LocalDataService.py

```diff
@@ -11,24 +11,20 @@
 from mantid.api import AlgorithmManager, mtd
 from pydantic import parse_file_as
 
-from snapred.backend.dao.calibration.Calibration import Calibration
-from snapred.backend.dao.calibration.CalibrationIndexEntry import CalibrationIndexEntry
-from snapred.backend.dao.calibration.CalibrationRecord import CalibrationRecord
-from snapred.backend.dao.GSASParameters import GSASParameters
-from snapred.backend.dao.InstrumentConfig import InstrumentConfig
-from snapred.backend.dao.Limit import Limit
-from snapred.backend.dao.ParticleBounds import ParticleBounds
-from snapred.backend.dao.RunConfig import RunConfig
-from snapred.backend.dao.state.CalibrantSample.CalibrantSamples import CalibrantSamples
-from snapred.backend.dao.state.DetectorState import DetectorState
-from snapred.backend.dao.state.DiffractionCalibrant import DiffractionCalibrant
-from snapred.backend.dao.state.FocusGroup import FocusGroup
-from snapred.backend.dao.state.InstrumentState import InstrumentState
-from snapred.backend.dao.state.NormalizationCalibrant import NormalizationCalibrant
-from snapred.backend.dao.StateConfig import StateConfig
-from snapred.backend.dao.StateId import StateId
+from snapred.backend.dao import GSASParameters, InstrumentConfig, Limit, ParticleBounds, RunConfig, StateConfig, StateId
+from snapred.backend.dao.calibration import Calibration, CalibrationIndexEntry, CalibrationRecord
```

Comment:
``` suggestion
from snapred.backend.dao.calibration import (
    Calibration,
    CalibrationIndexEntry,
    CalibrationRecord,
)
```

Reply:
I tried making this change, but pre-commit auto fixes to be one line again